collect_app/src/main/res/xml/other_preferences.xml

```diff
@@ -20,5 +19,4 @@
             android:layout="?android:attr/preferenceLayoutChild"
             android:title="@string/submission_url" />
     </PreferenceCategory>
-   
 </PreferenceScreen>
```

Comment:
I would add the newline here at the end of the file if for no other reason than to remove the warning.